torchsynth/module.py

```diff
@@ -227,7 +227,65 @@ def update_device(self, device=None):
             parameter.parameter_range.to(device)
 
 
-class ADSR(SynthModule):
+class ControlRateModule(SynthModule):
```

Comment:
Now you get me excited to add ParameterModule and Signal Module :)

Reply:
Right !? Like your comment below I like the idea of AudioRateModule and ParameterModule -- could be cool. It's actually giving me some ideas now ...